enterprise/__init__.py

```diff
@@ -4,6 +4,6 @@
 
 from __future__ import absolute_import, unicode_literals
 
-__version__ = '0.1.2'
+__version__ = '0.1.3'
```

Comment:
@mattdrayer @e-kolpakov what is the procedure for versioning and releasing? I think we should update this version only when we do a release (and might [update edx-platform](https://github.com/edx/edx-platform/pull/13917/files) correspondingly)